pkgs/development/python-modules/dask/default.nix

```diff
@@ -77,14 +77,6 @@ buildPythonPackage rec {
     "-m 'not network'"
   ];
 
-  disabledTests = [
-    "test_annotation_pack_unpack"
-    "test_annotations_blockwise_unpack"
-    # this test requires features of python3Packages.psutil that are
-    # blocked in sandboxed-builds
-    "test_auto_blocksize_csv"
```

Comment:
This test needs to be disabled on mac since it hits the `psutil` code giving this error:
```

    def auto_blocksize(total_memory, cpu_count):
        memory_factor = 10
>       blocksize = int(total_memory // cpu_count / memory_factor)
E       TypeError: unsupported operand type(s) for //: 'int' and 'NoneType'
```